spec/rspec/rails/example/view_example_group_spec.rb

```diff
@@ -90,6 +90,27 @@ module ::ApplicationHelper; end
       end
     end
 
+    describe "routes helpers collides with asset helpers" do
+      it "uses routes helpers" do
+        with_isolated_config do
+          ::Rails.application.routes.draw { resources :images }
+          output = nil
+
+          RSpec::Core::ExampleGroup.describe "a view spec", :type => :view do
```

Comment:
Rather than this, try to write this spec as below, where you create a class and include `ViewExampleGroup::ExampleMethods`.